src/main.js

```diff
@@ -1,23 +1,20 @@
-import 'bootstrap/dist/css/bootstrap.css'
-import 'bootstrap-vue/dist/bootstrap-vue.css'
+import './styles/legacy_bootstrap.scss'
+import './styles/index.css'
 
 import Vue from 'vue'
 import Vuex from 'vuex'
 import App from './App.vue'
+import VueRouter from 'vue-router'
 import router from './router'
 import store from './store'
 import globalAlert from '@/mixins/globalAlert'
-// Vue 2.7 (and 3) includes the composition API, so can remove this shortly.
-import VueCompositionAPI from '@vue/composition-api'
-import './styles/index.css'
```

Comment:
For once, a comment promising that something will happen 'shortly' turns out to be correct!